### PR TITLE
Closes #3035

### LINKED ISSUE #3035: Payout statements broken for Gemini creators
## Description

Currently, the payout statements section in the creators.brave.com dashboard only expects Uphold. If the Creator is connected to Gemini, the payout page displays a notice/error saying that there is an issue reading the transactions from Uphold, even though the Creator does not use Uphold. 

**Screenshot:**

image

### PR DESCRIPTION
**Pull request by tsmartt**
Closes #3035
For now we will not show this message if the user's selected wallet
is not an uphold wallet. If it is and we don't have the proper scope,
then show the message for reauthorization.

### CODE REVIEW

File: app/models/publisher.rb [hunk 1]
```diff
@@ -210,6 +210,10 @@ def sync_wallet_connections
     end
   end
 
+  def uphold_publisher?
```
Comment (yachtcaptain23) [inline app/models/publisher.rb:213]:
```suggestion
  def is_selected_wallet_provider_uphold?
```

Prefer renaming to something more understandable at-a-glance.

File: app/views/publishers/statements/index.html.slim [hunk 2]
```diff
@@ -9,31 +9,8 @@ noscript
   )
   = render partial: "choose_channel_button"
 
-/ .single-panel--wrapper.single-panel--wrapper--xlarge#statements_panel
```
Comment (yachtcaptain23) [inline app/views/publishers/statements/index.html.slim:12]: Would you see if `app/assets/stylesheets/pages/statements.scss` is removable too?

File: app/models/publisher.rb (hunk 1 above)
Comment (tsmartt) [reply app/models/publisher.rb]: Fixed.

File: app/views/publishers/statements/index.html.slim (hunk 2 above)
Comment (tsmartt) [reply app/views/publishers/statements/index.html.slim:12]: It was, fixed.

**Review by yachtcaptain23 — APPROVED** [review]

**Pull request merged**